vagrant-provision.sh

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
 
-# Install Apache and PHP (and any needed extensions).
-sudo yum install -y git php php-mcrypt php-pdo php-xml php-mbstring
+# Make sure apt has current list/updates
+sudo apt update
+sudo apt upgrade -y
+
+# Install necessary PHP building blocks
+# Install Apache and PHP (and any needed extensions). 
+# Install mock DB stuff
+sudo apt install -y zip unzip php php-dev php-pear \
```

Comment:
Why the need for `php-pear` and `php-dev`?

Reply:
Removed as suggested.